src/ShoppinPal/Vend/Api/V2/Products.php

```diff
@@ -68,4 +68,43 @@ public function get($productId)
         return new Product($result['data'], Product::UNKNOWN_PROPERTY_IGNORE, true);
 
     }
+
+    /**
+     * Uploads the give image of given product and
+     * Return response of product image upload.
+     * @param string $productId ID of the product.
+     * @param string $imageData of the image.
+     *
+     * @return data
```

Comment:
data type does not exist. @return should contain the exact type of the returned value in this case ProductImageUpload

Reply:
Added return type